cabal-install/Distribution/Client/SetupWrapper.hs

```diff
@@ -334,7 +335,8 @@ selfExecSetupMethod verbosity options _pkg bt mkargs = do
 
   searchpath <- programSearchPathAsPATHVar
                 (getProgramSearchPath (useProgramConfig options))
-  env        <- getEffectiveEnvironment [("PATH", Just searchpath)]
```

Comment:
I retract my comment. For some reason I thought that `act-as-setup --builddir` is a thing and was worried about potential inconsistencies.